spirv_msl.hpp

```diff
@@ -320,6 +333,18 @@ class CompilerMSL : public CompilerGLSL
 	void set_msl_options(const Options &opts)
 	{
 		msl_options = opts;
+		
+		// Provide the Metal bindings as part of the options structure as that is more convenient.
```

Comment:
Please remove this. This messes with the C API. There shouldn't be two paths to set this.